README.md

```diff
@@ -154,24 +142,12 @@ A list of companies that have paid Developer Community Writer Programs.
 - [PHP Architect](https://www.phparch.com/editorial/write-for-us/) - $175 per piece
   > Thought leadership and technical articles about PHP.
 
-- [Postmark](https://postmarkapp.com/write-for-us) - $200-$300 per piece
-  > Applications are currently closed.
-
 - [QuickNode](https://quicknode.notion.site/quicknode/QuickNode-Authorship-Program-d808a87ee50b48c9a16ed19b13e09115) - $350 per piece
   > Get paid to write articles about cryptocurrencies and web3/blockchain.
 
-- [Rancher](https://rancher.com/writing-program/roles/writer/) - $300 per piece
-  > Writing about devops, Kubernetes, and Rancher.
```

Comment:
Page 404'd.

I know Rancher got bought by Suse last year, so probably shut this down.